addons/mass_mailing/data/mass_mailing_data.xml

```diff
@@ -33,5 +33,144 @@
             <field name="email" model="res.users" eval="obj().env.ref('base.user_admin').email"/>
             <field name="list_ids" eval="[(6,0,[ref('mass_mailing.mailing_list_data')])]"/>
         </record>
+        <!-- Snippets' Default Images (to be replaced by themes) -->
+        <record id="mass_mailing.s_media_list_default_image_1" model="ir.attachment">
```

Comment:
Could you move to ir_attachment_data.xml file ? That way we split a bit data per model. Moreover if they are replaced by themes, let us not pollute other data.